src/main/java/core/basesyntax/DrawPrinter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawPrinter {

```

Comment:
Hint for naming interfaces in Java: https://www.oreilly.com/library/view/java-pocket-guide/9781491938683/ch01.html

"Interface names should be adjectives. They should end with “able” or “ible” whenever the interface provides a capability; otherwise, they should be nouns."

How about `Drawable` then?